keras_cv/models/convnext.py

```diff
@@ -386,6 +386,7 @@ def ConvNeXt(
 
 
 def ConvNeXtTiny(
+    *,
```

Comment:
Part of me wonders if, given the infrequency with which this language feature seems to be used, we should include a comment?